setup.py

```diff
@@ -9,6 +9,9 @@
     'pytest-runner',
     'boto3',
     'dcicutils',
+    'biopython',
+    'submit4dn',
```

Comment:
Why are you adding submit4dn to setup?  For xlrd and urlrequest?  May be better to add the specific modules rather than require Submit4dn?